src/Tasks/GenerateResource.cs

```diff
@@ -3444,18 +3460,23 @@ public static bool TryCreateCodeDomProvider(TaskLoggingHelper logger, string str
             {
                 provider = CodeDomProvider.CreateProvider(stronglyTypedLanguage);
             }
-#if FEATURE_SYSTEM_CONFIGURATION
```

Comment:
Elsewhere you still have configuration #if'ed out. Does CodeDomProvider actually throw configuration exception on .NET Core? I think we should avoid adding a dependency on it if it's not needed.